src/Element/TraversableElement.php

```diff
@@ -157,7 +157,11 @@ public function fillField($locator, $value)
             throw new ElementNotFoundException($this->getDriver(), 'form field', 'id|name|label|value|placeholder', $locator);
         }
 
-        $field->setValue($value);
+        if ($field->getTagName() === "select") {
```

Comment:
@aik099 I updated the pull request with the unit and driver test. Also
https://github.com/symfony/dom-crawler/blob/master/Field/ChoiceFormField.php#L120

does not accept label as an argument, only values. So which is it? driver should accept label as valid argument for `setValue` or Mink should use selectOption on 'select' nodes?